cns/restserver/restserver_test.go

```diff
@@ -689,3 +689,26 @@ func TestGetInterfaceForNetworkContainer(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestGetNumOfCPUCores(t *testing.T) {
+	fmt.Println("Test: getNumberOfCPUCores")
+
+	var err error
```

Comment:
Same comment for var ( as above. 
Move var w *httptest.ResponseRecorder up front

Reply:
Nopes, recorder will not be needed if the new request itself fails.